sympy/physics/quantum/cartesian.py

```diff
@@ -118,9 +116,7 @@ def _eval_hilbert_space(self, args):
     def _apply_operator_PxKet(self, ket):
         return ket.momentum*ket
 
-    def _represent_XKet(self, basis, **options):
-        index = options.pop("index", 1)
-
+    def _represent_XKet(self, basis, *, index=1, **options):
```

Comment:
I guess `**options` isn't needed any more

Reply:
I'm wary of touching this, I don't know what signature the caller expects `_represent_*` to have, so I'm inclined to leave this.